pkgs/top-level/python-packages.nix

```diff
@@ -14909,6 +14909,20 @@ in {
     };
   };
 
+  pint = buildPythonPackage rec {
+    name = "pint-${version}";
+    version = "0.7.2";
+
+    meta = {
+      description = "Physical quantities module";
+      license = licenses.bsd3;
+    };
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/c5/e5/1c317a30e51810d6ac1d744a6c232569c5a06d4478bdd20c2f0614d117e7/Pint-0.7.2.tar.gz";
```

Comment:
Sorry I forgot the uppercase for ”Pint” in the archive name. I have tested the following and it works perfectly:

```
$ nix-prefetch-url mirror://pypi/p/pint/Pint-0.7.2.tar.gz
downloading ‘https://pypi.io/packages/source/p/pint/Pint-0.7.2.tar.gz’... [18/146 KiB, 5.5 KiB/s]
path is ‘/nix/store/cis1svyzvs6jzgmf5417fpvxa8mkyypv-Pint-0.7.2.tar.gz’
1bbp5s34gcb9il2wyz4spznshahwbjvwi5bhjm7bnxk358spvf9q
```

Those urls are not the ones you find in the pypi webpage, but the work perfectly and do not need to be changed when the derivation is updated.